metadata.rb

```diff
@@ -7,7 +7,7 @@
 issues_url 'https://github.com/sous-chefs/issues'
 source_url 'https://github.com/sous-chefs/'
 chef_version '>= 12.5'
-version '3.0.0'
+version '3.1.0'
```

Comment:
Don't increment the version. 

```suggestion
version '3.0.0'
```